src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -166,6 +166,25 @@ const getLatestRelease = (distribution: Distributions) => {
     : distribution;
 };
 
+function removeShebangAndNewline(satelliteCommand: string) {
```

Comment:
I created this function so that when we revisit the wizard to edit it, we should not show to user that there is a shebang and newline.